packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx

```diff
@@ -86,7 +86,10 @@ export const WfoRelatedSubscriptions = ({
             field: 'description',
             name: t('description'),
             render: (value, record) => (
-                <Link href={`/subscriptions/${record.subscriptionId}`}>
+                <Link
+                    target="_blank"
+                    href={`/subscriptions/${record.subscriptionId}`}
```

Comment:
We hava a constants file with all app paths:
`packages/orchestrator-ui-components/src/components/WfoPageTemplate/paths.ts`

Also I see we did not always use that in the project, just een FYI

Reply:
ok, i will leave it as is.